### PR TITLE
added: startswith.md in csharp\concepts\strings\terms\startswith directory

### PR DESCRIPTION
**Pull request by kobrius11**
docs/content/c-sharp/concepts/strings/terms/startswith/startswith.md

### Description

Added new term StartsWith() in c-sharp concept entry. i used endswith.md as a reference, i dont know c-sharp but i used examples from endswith.md and i checked them, they work as intended. If change doesn't get pulled it was a pleasant experience, nonetheless. 

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [ ] All writings are my own.
- [+] My entry follows the Codecademy Docs style guide.
- [+] My changes generate no new warnings.
- [+] I have performed a self-review of my own writing and code.
- [+] I have checked my entry and corrected any misspellings.
- [+] I have made corresponding changes to the documentation if needed.
- [+] I have confirmed my changes are not being pushed from my forked `main` branch.
- [+] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [+] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/strings/terms/startswith/startswith.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+## Syntax
+
+```pseudo
+// Determines whether the String instance's first character matches the specified character
+String.StartsWith(char);
+
+// Determines whether the start of String instance matches the specified string
+String.StartsWith(string);
+
+// Same as above but uses additional case-sensitive and culture-sensitive
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/strings/terms/startswith/startswith.md:27]:
```suggestion
// Same as above but uses additional case-sensitive and culture-sensitive criteria.
```

File: content/c-sharp/concepts/strings/terms/startswith/startswith.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+ - 'learn-c-sharp'
+ - 'paths/computer-science'
+---
+
+The **`.StartsWith()`** method determines whether the start of the `string` instance matches the specified character or string. Additional parameters may be supplied to define the criteria for a match.
+
+## Syntax
+
+```pseudo
+// Determines whether the String instance's first character matches the specified character
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/strings/terms/startswith/startswith.md:21]:
```suggestion
// Determines whether the String instance's first character matches the specified character.
```

File: content/c-sharp/concepts/strings/terms/startswith/startswith.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+String.StartsWith(char);
+
+// Determines whether the start of String instance matches the specified string
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/strings/terms/startswith/startswith.md:24]:
```suggestion
// Determines whether the start of String instance matches the specified string.
```

File: content/c-sharp/concepts/strings/terms/startswith/startswith.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+String.StartsWith(string);
+
+// Same as above but uses additional case-sensitive and culture-sensitive
+String.StartsWith(string, ignoreCase, culture)
+
+// Same as above but uses an enumeration to determine the case- and culture-sensitive criteria
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/strings/terms/startswith/startswith.md:30]:
```suggestion
// Same as above but uses an enumeration to determine the case- and culture-sensitive criteria.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@kobrius11 I did a first review and left a few comments.

**Review by caupolicandiaz — APPROVED** [review]
@kobrius11 @SSwiniarski LGTM. I think this is good to merge.

**Comment by SSwiniarski** [comment]
@kobrius11 congrats on the new entry. Live link can be found here: https://www.codecademy.com/resources/docs/c-sharp/strings/startswith

You're currently listed as an anonymous contributor. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.

image

To add your GitHub username to your Codecademy profile:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

image

2.) Then log out and log back into Codecademy using your GitHub login:

image

**Note**: Your email used for Codecademy will have to be listed in your GitHub profile for the login feature to properly work. If you use the same email for both GitHub and Codecademy already, then you're set.

**Pull request merged**